packages/realm/src/app-services/SyncConfiguration.ts

```diff
@@ -208,11 +208,27 @@ export type BaseSyncConfiguration = {
   user: AnyUser;
   /**
    * Whether to create a new file and sync in background or wait for the file to be synced.
+   * The default behavior is:
+   *```
+   * {
+   *   type: OpenRealmBehaviorType.DownloadBeforeOpen,
+   *   timeOut: 30 * 1000,
+   *   timeOutBehavior: OpenRealmTimeOutBehavior.ThrowException,
+   * }
+   * ```
```

Comment:
I think this can work 🙂 
```suggestion
   * @default
   * {
   *   type: OpenRealmBehaviorType.DownloadBeforeOpen,
   *   timeOut: 30 * 1000,
   *   timeOutBehavior: OpenRealmTimeOutBehavior.ThrowException,
   * }
```